### PR TITLE
Komikku: update to 1.21.1

### PR DESCRIPTION
**Pull request by chrysos349**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-LIBC)

### CODE REVIEW

File: srcpkgs/Komikku/template [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+build_helper=gir
 hostmakedepends="gettext glib-devel gobject-introspection pkg-config
  desktop-file-utils gtk-update-icon-cache"
 makedepends="gtk4-devel libadwaita-devel"
 depends="gtk4 libadwaita libnotify libsecret python3-BeautifulSoup4 python3-Brotli
  python3-Pillow python3-Unidecode python3-requests python3-dateparser
  python3-gobject python3-keyring python3-lxml python3-magic python3-rarfile
  python3-natsort python3-pure-protobuf python3-emoji libwebkitgtk60
- python3-piexif"
+ python3-piexif bubblewrap xdg-dbus-proxy"
```
Comment (paper42) [inline srcpkgs/Komikku/template:14]: why is this necessary?

Comment (paper42) [reply srcpkgs/Komikku/template:14]: I think this is a problem with your setup and should be left to you to solve. I don't see any mention of bubblewrap in the source code.

Comment (chrysos349) [reply srcpkgs/Komikku/template]: Reverted.
